### ISSUE TITLE
GF_Field::get_value_entry_detail was called incorrectly error

### ISSUE THREAD

**Issue by jakejackson1** [GravityPDF/gravity-pdf#568]
`GF_Field::get_value_entry_detail was called <strong>incorrectly</strong>. Override this
method to handle array values Please see <a
href="https://codex.wordpress.org/Debugging_in_WordPress">Debugging in WordPress</a> for more
information. (This message was added in version 2.0.) in /srv/www/wordpress-default/wp-
includes/functions.php on line 
4091`

**Comment by jakejackson1** [comment]
Wasn't able to replicate in Zadani using all core, survey, poll, quiz or signature fields using the latest Gravity Forms and WordPress.

**Comment by jakejackson1** [comment]
![hidden-production-error](https://cloud.githubusercontent.com/assets/2918419/22225201/a774cd36-e215-11e6-8787-6df72404e8ab.png)

**Comment by jakejackson1** [comment]
Did additional testing and haven't been able to replicate the problem yet.

**Comment by mlp** [comment]
I've had this come up today, well, over the last few days but that $prev_reporting_level = error_reporting( 0 ); thing was an interesting pita that has made this issue span days. Anyways, I'll report back if I figure something out.

**Comment by TriptiSharma** [comment]
Hi,

I am getting the same thing:
Notice: GF_Field::get_value_entry_detail was called incorrectly. Override this method to handle array values Please see Debugging in WordPress for more information. (This message was added in version 2.0.) in /var/www/html/pci/wp-includes/functions.php on line 4161

Did anyone found a solution to this one?

**Comment by TriptiSharma** [comment]
@jakejackson1 I have been able to replicate the issue. Here is the plugin the I am working on:
https://github.com/BellevueCollege/gf-sqlserver-integration

In File gf-sqlserver-integration.php --- Function gfsi_after_submission
line 170  $this_form = GFAPI::get_form(rgar($entry, 'form_id'));  
generates the Notice that I mentioned in the previous comment. 
Here it didn't allow me to attach the .json file form export. So I changed the extension to .txt.
If you want to import the form you might want to change the extension to .json.

[gravityforms-export-2018-10-11.txt](https://github.com/GravityPDF/gravity-pdf/files/2470888/gravityforms-export-2018-10-11.txt)

The plugin is easy to setup and basically exports the entries to MSSQL database on submit of the form.
Let me know if you have any questions.
Since I have not been able to find any other posts online on this one, any help would be greatly appreciated.

Thanks,
Tripti
